### ISSUE TITLE
How should `edward2` be installed?

### ISSUE THREAD

**Issue by batzner** [google-research/robustness_metrics#2]
I noticed this import in `robustness_metrics/models/uncertainty_baselines.py`:

https://github.com/google-research/robustness_metrics/blob/b4c4d4d19e3ef0f2dde5aa44292f82eddeb86906/robustness_metrics/models/uncertainty_baselines.py#L28

However, [`edward2/experimental`](https://github.com/google/edward2/tree/master/experimental) is not covered by the the `find_packages()` call in [`edward2/setup.py`](https://github.com/google/edward2/blob/master/setup.py), meaning that the `experimental` directory will not be installed by pip.

Therefore, I tried simply cloning the edward2 repository and setting the PYTHONPATH to its parent directory, such that `import edward2.experimental` works. This unfortunately caused these imports in [`edward2/experimental/sngp/__init__.py`](https://github.com/google/edward2/blob/master/experimental/sngp/__init__.py) to fail:

```
from experimental.sngp.gaussian_process import *  # local file import
from experimental.sngp.normalization import *  # local file import
from experimental.sngp.utils import *  # local file import
```

These require `experimental` to be discoverable by the PYTHONPATH, but the path needs to be set to the parent directory of edward2 to make the `from edward2.experimental import sngp` import above work.

So, I am wondering what the recommended way to import `edward2.experimental` is and how you do it yourselves?

**Comment by dustinvtran** [comment]
Good catch. It looks like a mistake that this codebase imports from edward2.experimental. We purposefully don't include that code into the library. I'll drop that dependence.
